### DISCUSSION TITLE
Задание 8

### DISCUSSION THREAD

**Discussion by eprilutskaya** [anastasiaprakapchuk/AmastyLearningNewTechnology#24] [General]
https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/ff1eeeee5c4d3813c60aa17764c6b6681d950036/AnastasiaModule/Model/BlacklistRepository.php#L38
https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/ff1eeeee5c4d3813c60aa17764c6b6681d950036/AnastasiaModule/Cron/AmNotification.php#L47
Если вам нужно получить значение из записи, то для этого не нужно реализовывать методы в репозитории, это можно получить из объекта модели и методы должны называться соответствующе getSku() , getQty()
```
...
'blacklistItemSku' => $blacklistItem->getSku(),
'blacklistItemQty' => $blacklistItem->getQty()
...
```
getBy - используется для методо получения объектов по некторому параметру
getBySku()- получить что-либо по ску
 
https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/ff1eeeee5c4d3813c60aa17764c6b6681d950036/AnastasiaModule/Cron/AmNotification.php#L69
здесь уже есть объект $blacklistItem , у которого можно выполнить setData и сохранить через blacklistResource (добавить в конструктор или реализовать просто метод save в репозитории , где доступно blacklistResource)
```
$blacklistItem->setData('email_body', $emailBody);

```
Оценка 9

**Comment by anastasiaprakapchuk** [comment]
Cпасибо, я исправила замечания
